config/ozu-client.php

```diff
@@ -7,6 +7,9 @@
         //        \App\Models\Project::class,
     ],
 
+    // You can configure Site Settings and administrate them from Ozu by setting your settings class here (see documentation).
```

Comment:
Mmm, better comment:

The \Code16\OzuClient\Support\Settings\OzuSiteSettings implementation to manage your site general settings